### PR TITLE
Refactor naming generation

### PR DESCRIPTION
**Pull request by VeroGarp**
## Description
Refactor naming generation

## Changes
Add a new function in a new file

## Checks
- [x] Project Builds
- [x] Project passes tests and checks
- [ ] Updated documentation accordingly

### CODE REVIEW

**Review by NickSeagull — APPROVED** [review]
LGTM, thanks!

**Comment by NickSeagull** [comment]
Can you please rebase your branch to get this merged after integration tests? Thanks 🙏

**Comment by VeroGarp** [comment]
Done :)

**Comment by NickSeagull** [comment]
integration: all

**Review by adriangmweb — APPROVED** [review]
Thanks again for your work, LGTM!

File: packages/cli/src/common/filenames.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import * as inflected from 'inflected'
+
+
+export function classNameToFileName(name:string): string {
```
Comment (javiertoledo) [inline packages/cli/src/common/filenames.ts:4]: Good idea!

**Review by javiertoledo — APPROVED** [review]
🚢 🇮🇹 ❗

**Comment by NickSeagull** [comment]
Merging as integration tests are still failing in master...

**Pull request merged**